tests/test_arrow.py

```diff
@@ -504,6 +489,7 @@ def test_strftime(self):
         assert result == self.arrow._datetime.strftime("%Y")
 
 
+@pytest.mark.usefixtures("timedst_fixture")
```

Comment:
Since this fixture is not used anywhere else and represents a special case, let's move the variables that are currently exposed in the fixture into local variables in the function.

Reply:
got it.